src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -13,7 +13,8 @@
      * <p>Это задание необходимо решать с использованием логических операторов,
      * без использования конструкций if-else</p>
      */
+
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return ((a ^ b) & (c ^ d)) | ((a ^ c) & (b ^ d)) | ((a ^ d) & (c ^ b));
```

Comment:
there are redundant checks here, try to simplify it

Reply:
Don't see any
How is that possible if I check all the possible combinations? Or where is redundancy?

Reply:
you should check all unique cases, there are some cases that is cover by others(so they are redundant)